uui-components/src/layout/ScrollBars.tsx

```diff
@@ -71,7 +71,7 @@ export const ScrollBars = forwardRef<ScrollbarsApi, ScrollbarProps>(({
             hideTracksWhenNotNeeded
             ref={ bars }
             { ...props }
-            { ...rawProps }
+            { ...(rawProps as unknown as LibScrollbarProps) }
```

Comment:
`LibScrollbarProps` wrongly typed in the library itself,
 
`HTMLAttributes<Scrollbars>` incompatible with `HTMLAttributes<HTMLDivElement>` - it has conflicts in DOM methods, since `Scrollbars` is a React component type, and not inherits `HTMLElement`

So, we should locally patch package or transform type this way by force